x-pack/plugin/eql/qa/correctness/build.gradle

```diff
@@ -28,6 +28,9 @@ testClusters {
     if (serviceAccountFile) {
       keystore 'gcs.client.eql_test.credentials_file', serviceAccountFile
     }
+    if (Boolean.parseBoolean(System.getProperty('eql.test.preserve.data', 'false'))) {
```

Comment:
Can we use `provider.systemProperty(...)` here to slowly move towards gradle configuration cache compliant builds? System.getProperty isn't

Reply:
Good call. I've updated the logic for finding the service account file as well.